7주차/[Lv.2]_테이블_해시_함수/안혜수.js

```diff
@@ -0,0 +1,14 @@
+function solution(data, col, row_begin, row_end) {
+  const answer = [];
+  col -= 1;
+
+  data.sort((a, b) => {
+    if (a[col] === b[col]) return b[0] - a[0];
+    return a[col] - b[col];
+  });
+
+  for (let i = row_begin; i < row_end + 1; i++) {
+    answer.push(data[i - 1].reduce((acc, cur) => (acc += cur % i), 0));
+  }
+  return answer.reduce((acc, cur) => (acc ^= cur), 0);
```

Comment:
```suggestion
  return answer.reduce((acc, cur) => acc ^ cur, 0);
```

반환하는 값이 다음 `acc`이기 때문에 `=`는 없애는 것이 더 명확할 것 같습니다!! 🤔